src/presets.rs

```diff
@@ -35,17 +33,16 @@ pub fn generate_safe_prime<T: Integer + RandomBits + RandomMod>(
     generate_safe_prime_with_rng(&mut OsRng, bit_length, bits_precision)
 }
 
-/// Checks probabilistically if the given number is prime using [`OsRng`] as the RNG.
+/// Probabilistically check if the given number is prime using [`OsRng`] as the RNG.
```

Comment:
I think elsewhere in the library we use third person instead of imperative. I don't mind either way, but we better be consistent throughout.

Reply:
Good point, I'll switch back to third person.